et/docker.ts

```diff
@@ -156,7 +162,7 @@ function ccdComposePull() {
 
     temporaryLog(`Running ${command}`)
 
-    const child = exec(command, { cwd: process.env.ECM_DOCKER_DIR }, (err => {
+    const child = exec(command, { cwd: process.env.ECM_DOCKER_DIR, env: { ...process.env, ...getEnvVarsFromFile() } }, (err => {
```

Comment:
Why doesn't this use the executeCommand helper method?

Reply:
the execCommand helper method will only return when execution is finished, we want to monitor stdout and stderr during this longer running command